Library/Homebrew/os/mac/xcode.rb

```diff
@@ -27,6 +27,11 @@ def latest_version
         end
       end
 
+      def prerelease?
+        # TODO: bump version after Xcode 8.0 is stable.
+        version > "7.3.1"
```

Comment:
Maybe make that `version >= "8.0"` in case we'll be graced with Xcode 7.4 (though by now it seems unlikely)? What would we bump this to once Xcode 8 becomes stable?